source/index.html.md

```diff
@@ -579,7 +597,7 @@ This <code>wager_id</code> is used in subsequent calls when required as part of
 | HTTP Status Code | Error Description |
 | ---| --- |
 | `201 Created` | Success
-| `208 Already Reported` | Success, if the `external_wager_id` was already processed |
+| `208 Already Reported` | Success, the wager with `external_wager_id` was already created |
```

Comment:
So for the `Create X` calls does it need both `208` and `409`? The `208` is if you use idempotency headers, the `409` is if you don't?